src/applications/disability-benefits/all-claims/tests/toxic-exposure.cypress.spec.js

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import { cloneDeep } from 'lodash';
+import mockUser from './fixtures/mocks/user.json';
+import mockFeatureToggles from './fixtures/mocks/feature-toggles.json';
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+import { setup, pageHooks } from './cypress.helpers';
+
+const toggles = cloneDeep(mockFeatureToggles);
+toggles.data.features.push({
+  name: 'disability_526_toxic_exposure',
+  value: true,
+});
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+
+    // run the newOnly-test to verify enabling TE doesn't break things. then also run maximal using all TE pages
+    dataSets: ['newOnly-test', 'maximal-toxic-exposure-test'],
+
+    fixtures: {
+      data: path.join(__dirname, 'fixtures', 'data'),
+    },
+
+    pageHooks: pageHooks(cy, toggles),
+    setupPerTest: () => {
+      cy.login(mockUser);
+      setup(cy, toggles);
+    },
+
+    useWebComponentFields: true,
+
+    // skip: [],
+  },
+  manifest,
+  formConfig,
+);
+
+testForm(testConfig);
```

Comment:
@aurora-a-k-a-lightning i was thinking about your point about adding a negative test. I looked through the existing all-claims e2e tests and it doesn't look like we have a test for CFI only. I imagine this is a common scenario we should cover and in that flow, TE pages _should not_ display. 

If we think this is a good idea, I can add it in. However, given how much time I've already spent on this story, I'm inclined to create a separate story for that. lmk wdyt 😄

Reply:
@christinec-fftc separate story sounds good! 👍